ToDoList/src/ToDoList.API.DependencyInjection/DependencyBootstraper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Web.Http.Dependencies;
+
+using ToDoList.Contracts;
+using ToDoList.DependencyInjection;
+
+namespace ToDoList.API.DependencyInjection
+{
+    public class DependencyBootstraper
+    {
+        private readonly Container _container = new Container();
+        
+        public void RegisterDependencies()
+        {
+            Register<Repository.DependencyRegister>(_container);
+            Register<Services.DependencyRegister>(_container);
+        }
+
+        public void RegisterApiSingleton<TFrom, TTo>()
```

Comment:
As mentioned at [this comment](https://github.com/vaclavm/kentico-onboarding-cs/pull/6/files#diff-edee51a4cbe7f75e88f4ebb21a636091R15), there might be just `TFrom` generic and a `TFrom` or `Func<TFrom>` as a parameter…

Reply:
Solve by registering instance instead of singleton